src/FixedDataTable.js

```diff
@@ -308,19 +305,6 @@ var FixedDataTable = React.createClass({
   componentWillMount() {
     const props = this.props;
 
-    const {
-      columnGroups,
-      elementTemplates,
```

Comment:
moved elementTemplates to state for the time being.
Any reason to avoid this @wcjordan ?